services/retrieve_and_rank/v1.js

```diff
@@ -419,12 +424,9 @@ module.exports = function (RED) {
     });
   }
 
-  RED.nodes.registerType('watson-retrieve-rank-create-cluster', createClusterNode, {
-    credentials: {
-      username: {type:"text"},
-      password: {type:"password"}
-    }
-  });
+  RED.nodes.registerType("watson-retrieve-rank-credentials", serviceCredentialsConfigurationNode);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2647159221/issues/source?bid=3225115&fileBranchId=3322820#l427)